src/zcl_abapgit_stage_logic.clas.abap

```diff
@@ -87,9 +87,9 @@ CLASS ZCL_ABAPGIT_STAGE_LOGIC IMPLEMENTATION.
 
   METHOD zif_abapgit_stage_logic~get.
 
-    rs_files-local  = io_repo->get_files_local( ).
-    rs_files-remote = io_repo->get_files_remote( ).
-    rs_files-status = io_repo->status( ).
+    rs_files-local  = io_repo->get_files_local( ii_pre_filter = ii_pre_filter ).
+    rs_files-remote = io_repo->get_files_remote( ii_pre_filter = ii_pre_filter ).
+    rs_files-status = io_repo->status( ii_pre_filter = ii_pre_filter ).
```

Comment:
https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md#omit-the-parameter-name-in-single-parameter-calls